layouts/community/ergodox/colemak_code_friendly/keymap.c

```diff
@@ -0,0 +1,328 @@
+ * | Grave  |   !  |   @  |   #  |   $  |   %  |   ^  |  |      |      |   *  |   (  |  )   |  -   |        |
+ * |--------+------+------+------+------+------+------|  |------+------+------+------+------+------+--------|
+ * | Tab    |   Q  |   W  |   F  |   P  |   G  |   =  |  |  =   |  =>  |   7  |   8  |   9  |  +   |        |
+ * |--------+------+------+------+------+------|      |  |      |------+------+------+------+------+--------|
+ * | Esc    |   A  |   R  |   S  |   T  |   D  |------|  |------|  ->  |   4  |   5  |   6  |  0   |        |
+ * |--------+------+------+------+------+------|   {  |  | Enter|------+------+------+------+------+--------|
+ * | LShift |Z/LCtl|X/Supe|   C  |   V  |B/RAlt|      |  |      |  !=  |   1  |   2  |   3  |      |        |
+ * '--------+------+------+------+------+-------------'  '-------------+------+------+------+------+--------'
+ *   | LCtl |Super | Alt  |   [  |   ]  |                              |   0  |   ,  |   .  |      |      |
+ *   '----------------------------------'                              '----------------------------------'
+ *                                      .-------------.  .-------------.
+ *                                      | Home | End  |  |      | Stop |
+ *                               .------+------+------|  |------+------+------.
+ *                               |      |      |PrnScr|  |Start2|      |      |
+ *                               |CapsLk| Enter|------|  |------|      |      |
+ *                               |      |      | Super|  |Play2 |      |      |
+ *                               '--------------------'  '--------------------'
+ */
+[LAYER_RNUM] = LAYOUT_ergodox(
+  /* left hand */
+  S(KC_GRV),   KC_EXLM,     KC_AT,       KC_HASH,     KC_DLR,      KC_PERC,    KC_CIRC,
+  S(KC_TAB),   S(KC_Q),     S(KC_W),     S(KC_F),     S(KC_P),     S(KC_G),    S(KC_EQL),
+  S(KC_ESC),   S(KC_A),     S(KC_R),     S(KC_S),     S(KC_T),     S(KC_D),
+  KC_LSFT,     S(KC_Z),     S(KC_X),     S(KC_C),     S(KC_V),     S(KC_B),    KC_LCBR,
+  S(KC_LCTL),  S(KC_LGUI),  S(KC_LALT),  S(KC_LBRC),  S(KC_RBRC),
+  
+                                                              S(KC_HOME), S(KC_END),
+                                                                         S(KC_PSCR),
+                                                   KC_CAPS, S(KC_ENT),   S(KC_LGUI),
+  /* right hand */
+  KC_TRNS, KC_TRNS,     KC_ASTR, KC_LPRN,  KC_RPRN,  KC_MINS,  KC_TRNS,
+  KC_EQL,  M_LAMBDA,    KC_7,    KC_8,     KC_9,     KC_PLUS,  KC_TRNS,
+           M_POINER,    KC_4,    KC_5,     KC_6,     KC_0,     KC_TRNS,
+  KC_ENT,  M_NOT_EQL,   KC_1,    KC_2,     KC_3,     KC_TRNS,  KC_TRNS,
+                        KC_0,    KC_COMM,  KC_DOT,   KC_TRNS,  KC_TRNS,
+  KC_TRNS, DYN_REC_STOP,
+  DYN_REC_START2,
+  DYN_MACRO_PLAY2, KC_TRNS, KC_TRNS
+),
+};
+
+const uint16_t PROGMEM fn_actions[] = {
```

Comment:
Lines 207 and 208 should be deleted; that function is deprecated, and your keymap doesn't use it anyway.

Reply:
Removed, thanks!